internal/db/storage/blob/s3/writer/writer_test.go

```diff
@@ -51,36 +61,44 @@ func TestNew(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           opts: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           opts: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns writer when option is empty",
+			args: args{
+				opts: nil,
+			},
+			want: want{
+				want: &writer{
+					eg:          errgroup.Get(),
+					contentType: "application/octet-stream",
+					maxPartSize: 64 * 1024 * 1024,
+					s3manager:   s3manager.New(),
+				},
+			},
+		},
+
+		{
+			name: "returns writer and outputs the warn when option is not empty",
```

Comment:
🤔 it is difficult to check if the warning message is outputted or not 🤣

Reply:
I think so too.
If the writer returns an error, I will fix it